lib/package.py

```diff
@@ -89,19 +89,20 @@ def __repr__(self):
         return self.name
 
 
-    def download_files(self):
+    def download_files(self, recurse=True):
```

Comment:
No need, if you're right you're right. I swear I translated it before being pedantic =).. +1